docs/source/gs-processing/usage/example.rst

```diff
@@ -211,26 +211,41 @@ and can be used downstream to create a partitioned graph.
 .. code-block:: bash
 
     $ cd /tmp/gsprocessing-example
-    $ ls
-
-    edges/  launch_arguments.json  metadata.json  node_data/
-    raw_id_mappings/  perf_counters.json  updated_row_counts_metadata.json
+    $ ls -l
+
+    edges/
+    gconstruct-config_with_transformations.json
```

Comment:
Agree on the confusing part, but since we want to support cross-compatibility with GConstruct, it's the least invasive solution. 

Otherwise we'd have to re-parse the entire config file and do that for both versions of the configs. 

I'd prefer to keep it this way, one is intended for end users (`gconstruct-config_with_transformations.json`) the other just to be read by GSProcessing.

Reply:
OK